Automattic/wp-calypso
# Publicize scheduling: seperate published endpoint

Use new endpoint to seperate scheduled from published.
This one: D4834-code

## Testing instructions

Go to post -> share, you should see this:
<img width="607" alt="zrzut ekranu 2017-03-22 o 11 06 04" src="https://cloud.githubusercontent.com/assets/3775068/24207717/990522ee-0eef-11e7-8fbc-870474f5829d.png">


CC @lamosty @retrofox
